client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -198,49 +198,31 @@ class PreviewFrame extends React.Component {
     this.resolveScripts(sketchDoc, resolvedFiles);
     this.resolveStyles(sketchDoc, resolvedFiles);
 
-    let scriptsToInject = [
+    const scriptsToInject = [
       '/loop-protect.min.js',
       '/hijackConsole.js'
     ];
-    if (
-      this.props.isAccessibleOutputPlaying ||
-      ((this.props.textOutput || this.props.gridOutput || this.props.soundOutput) && this.props.isPlaying)) {
-      let interceptorScripts = [];
-      interceptorScripts = [
-        '/p5-interceptor/registry.js',
-        '/p5-interceptor/loadData.js',
-        '/p5-interceptor/interceptorHelperFunctions.js',
-        '/p5-interceptor/baseInterceptor.js',
-        '/p5-interceptor/entities/entity.min.js',
-        '/p5-interceptor/ntc.min.js'
-      ];
-      if (!this.props.textOutput && !this.props.gridOutput && !this.props.soundOutput) {
-        this.props.setTextOutput(true);
-      }
-      if (this.props.textOutput) {
-        let textInterceptorScripts = [];
-        textInterceptorScripts = [
-          '/p5-interceptor/textInterceptor/interceptorFunctions.js',
-          '/p5-interceptor/textInterceptor/interceptorP5.js'
-        ];
-        interceptorScripts = interceptorScripts.concat(textInterceptorScripts);
-      }
-      if (this.props.gridOutput) {
-        let gridInterceptorScripts = [];
-        gridInterceptorScripts = [
-          '/p5-interceptor/gridInterceptor/interceptorFunctions.js',
-          '/p5-interceptor/gridInterceptor/interceptorP5.js'
-        ];
-        interceptorScripts = interceptorScripts.concat(gridInterceptorScripts);
-      }
-      if (this.props.soundOutput) {
-        let soundInterceptorScripts = [];
-        soundInterceptorScripts = [
-          '/p5-interceptor/soundInterceptor/interceptorP5.js'
-        ];
-        interceptorScripts = interceptorScripts.concat(soundInterceptorScripts);
-      }
-      scriptsToInject = scriptsToInject.concat(interceptorScripts);
+    const accessiblelib = sketchDoc.createElement('script');
+    accessiblelib.setAttribute('src', 'https://cdn.rawgit.com/MathuraMG/p5-accessibility/e856365c/dist/p5-accessibility.js');
```

Comment:
rather than link to the accessibility using raw git, i think it would be better to add the library as a dependency in `package.json`, then import the built script using webpack.

Reply:
cc @lm-n, @cleezyITP - We have been talking about this one, we would like the library to be a part of the HTML code as a CDN so that when a user shares their code (say, send a zip to someone, it will include the library. This would make sharing more accessible. What do you think?

Reply:
@catarak - was wondering what your thoughts on this are?